pulsar/producer_partition.go

```diff
@@ -441,30 +446,37 @@ func (p *partitionProducer) reconnectToBroker() {
 		}
 		p.log.WithError(err).Error("Failed to create producer at reconnect")
 		errMsg := err.Error()
-		if strings.Contains(errMsg, errTopicNotFount) {
+		if strings.Contains(errMsg, errMsgTopicNotFound) {
 			// when topic is deleted, we should give up reconnection.
-			p.log.Warn("Topic Not Found.")
+			p.log.Warn("Topic not found, stop reconnecting, close the producer")
+			// in JAVA client, when topic not found, closeAsync() will be called, see
+			// nolint: lll
+			// https://github.com/apache/pulsar/blob/master/pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerImpl.java#L1792-L1799
```

Comment:
Maybe we can remove these comments? It's good for review. But it looks redundant as comments.

Reply:
OK, I have deleted these comments.